index.html

```diff
@@ -206,8 +217,10 @@ <h4>Skills</h4>
       </ul>
     </section>
     <footer class="contact-group">
-      <div class="bgft">
+      
       <form action="index.html" method="post">
+        <div class="bgft">
+       <div class="arrange-form">
       <div class="contact-me">
         <h2 class="contact_align">Contact me</h2>
         <p class="pitch">
```

Comment:
- [ ] Kindly be noticed that the form should be wider in the desktop size.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-03 01-39-56](https://user-images.githubusercontent.com/33752735/171758603-f46cfa20-599c-4415-a0f9-03975fe416fe.png) | ![Screenshot from 2022-06-03 01-39-37](https://user-images.githubusercontent.com/33752735/171758620-3a1b5618-7834-4809-9d80-6ff8a1181aa6.png)